examples/particles/index.html

```diff
@@ -0,0 +1,303 @@
+<html>
+
+  <head>
+    <title>Particle demo</title>
+    <meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+    <link href="../examples.css" type="text/css" rel="stylesheet" media="screen" />
+    <style>
+      body {
+        margin: 0px;
+        background: #000;
+      }
+      #render-canvas {
+        width: 100%;
+        height: 100%;
+      }
+    </style>
+
+    <script src="http://d1n0x3qji82z53.cloudfront.net/src-min-noconflict/ace.js" type="text/javascript" charset="utf-8"></script>
+    <script type="text/javascript" src="../../dist/LumaGL.js"></script>
+    <script type="text/javascript" src="../examples.js"></script>
+    <script type="text/javascript" src="index.js"></script>
```

Comment:
Not sure you need the `type` attributes on here; could be argued the `charset` attribute could be left off as well.